ts/features/profile/components/Error.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { H3, Pictogram, VSpacer } from "@pagopa/io-app-design-system";
+import { View } from "react-native";
+import { IOStyles } from "../../../components/core/variables/IOStyles";
+
+export const ErrorComponent = () => (
+  <View style={[IOStyles.flex, IOStyles.centerJustified, IOStyles.alignCenter]}>
+    <Pictogram name="attention" size={72} />
+    <VSpacer size={16} />
+    <H3>We could not get your profile :(</H3>
```

Comment:
What do you think to use standard locales instead inline string? You can add new strings under `locales/it/index.yml`